GoogleSignIn/Sources/GIDAppCheck/Implementations/Fake/GIDAppCheckProviderFake.m

```diff
@@ -22,14 +22,14 @@
 
 @interface GIDAppCheckProviderFake ()
 
-@property(nonatomic, strong, nullable) id<GACAppCheckTokenProtocol> token;
+@property(nonatomic, strong, nullable) GACAppCheckToken *token;
```

Comment:
I've updated the signatures, but I would like this to be nullable since I will create instances of this fake with no token in scenarios where I want AppCheckCore to give me the placeholder token.